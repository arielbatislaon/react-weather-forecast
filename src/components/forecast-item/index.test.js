import React from 'react';
import renderer from 'react-test-renderer';

import ForeCastItem from './index';
const forecast ={};

it('renders ForeCastItem correctly ', () => {
  const tree = renderer.create(<ForeCastItem forecast={forecast}/>).toJSON();
  expect(tree).toMatchSnapshot();
});