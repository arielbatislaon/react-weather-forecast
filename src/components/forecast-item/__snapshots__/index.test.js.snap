// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ForeCastItem correctly  1`] = `
<div
  className="col"
>
  <div>
    <h5>
      Wednesday
    </h5>
  </div>
  <div>
    Min Temp.
    :
  </div>
  <div>
     
    Max Temp.
    :
  </div>
</div>
`;
